guides/common/modules/con_scenarios-for-foreman-on-aws.adoc

```diff
@@ -0,0 +1,84 @@
+[id="scenarios-for-{project-context}-on-aws"]
+= Scenarios for {Project} on AWS
+
+There are several deployment scenarios for {Project} on Amazon Web Services (AWS):
+
+* One-region setup
+* Connecting on-premises and AWS region
+* Connecting different regions
+
```

Comment:
```suggestion
```
Should be safe to drop, right? It's a maintenance risk that doesn't really bring that much value.